source/assets/css/index.css

```diff
@@ -9,13 +9,14 @@
 }
 
 body {
-    background: -webkit-linear-gradient(left, #25c481, #25b7c4);
-    background: linear-gradient(to right, #25c481, #25b7c4);
+    /* background: -webkit-linear-gradient(left, #25c481, #25b7c4);
+    background: linear-gradient(to right, #25c481, #25b7c4); */
+    background-color: whitesmoke;
```

Comment:
Not a fan of manually specifying colors like this. What if we add dark mode? We need a design person to make a sheet of all the designated colors and make a file that specifies them all as constants. @nathanueda @y1qu

Reply:
@Samir-Rashid I like that idea, just let me know if you want to definitely move ahead with this and i'll put it together.